Parse/src/main/java/com/parse/ParseLogInterceptor.java

```diff
@@ -240,19 +310,34 @@ private void logRequestInfo(final String requestId,
     logger.unlock();
   }
 
-  private String getRequestBodyInfo(ParseHttpRequest request)
-      throws IOException {
-    if (request.getBody() == null) {
-      return null;
-    }
+  // Since we can not read the content of the response directly, we need an additional parameter
+  // to pass the responseBody after we get it asynchronously.
+  private void logResponseInfo(
+      Logger logger, String requestId, ParseHttpResponse response, String responseBodyInfo) {
+    logger.lock();
+    logger.writeLine(KEY_TYPE, TYPE_RESPONSE);
+    logger.writeLine(KEY_REQUEST_ID, requestId);
+    logger.writeLine(KEY_STATUS_CODE, String.valueOf(response.getStatusCode()));
+    logger.writeLine(KEY_REASON_PHASE, response.getReasonPhrase());
+    logger.writeLine(KEY_HEADERS, response.getAllHeaders().toString());
 
-    String requestContentType = request.getBody().getContentType();
-    if (isContentTypePrintable(requestContentType)) {
-      ByteArrayOutputStream output = new ByteArrayOutputStream();
-      request.getBody().writeTo(output);
-      return formatBytes(output.toByteArray(), requestContentType);
-    } else {
-      return IGNORED_BODY_INFO;
+    // Body
+    if (responseBodyInfo != null) {
+      logger.writeLine(KEY_BODY, responseBodyInfo);
```

Comment:
e... Why? If `responseBodyInfo` is not null, we should just print `responseBodyInfo`.